envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/ChaosControllerTest.kt

```diff
@@ -97,6 +97,76 @@ internal class ChaosControllerTest : EnvoyControlTestConfiguration() {
         assertThat(response.experimentList).isEqualTo(emptyList<NetworkDelayResponse>())
     }
 
+    @Test
+    fun `should follow the flow`() {
```

Comment:
Test name should describe, what it actually do

Reply:
test has been changed